tests/php/test-class-amp-meta-box.php

```diff
@@ -158,20 +158,34 @@ public function test_enqueue_block_assets() {
 		);
 		$this->assertEquals( AMP_Post_Meta_Box::BLOCK_ASSET_HANDLE, $block_script->handle );
 		$this->assertEquals( amp_get_asset_url( 'js/' . AMP_Post_Meta_Box::BLOCK_ASSET_HANDLE . '.js' ), $block_script->src );
-
-		$this->assertContains( 'ampBlockEditor', $block_script->extra['data'] );
+		/**
+		 * @since 2.0.9
+		 * Values are now loaded using wp_inline_script()
+		 */
+		$data = $block_script->extra['before'][1];
+		$this->assertContains( 'ampBlockEditor', $data );
 		$expected_localized_values = [
 			'ampSlug',
 			'errorMessages',
 			'hasThemeSupport',
 			'isStandardMode',
+			'featuredImageMinimumHeight',
+			'featuredImageMinimumWidth',
 		];
 
 		foreach ( $expected_localized_values as $localized_value ) {
-			$this->assertContains( $localized_value, $block_script->extra['data'] );
+			$this->assertContains( $localized_value, $data );
 		}
 	}
 
+	public function test_featured_image_dimensions() {
+		$dimensions = AMP_Post_Meta_Box::get_featured_image_dimensions();
+		$this->assertArrayHasKey( 'featuredImageMinimumHeight', $dimensions );
+		$this->assertArrayHasKey( 'featuredImageMinimumWidth', $dimensions );
+		$this->assertEquals( $dimensions['featuredImageMinimumWidth'], 1200 );
+		$this->assertEquals( $dimensions['featuredImageMinimumHeight'], 675 );
```

Comment:
Please also add `amp_featured_image_minimum_width` and `amp_featured_image_minimum_height` filters, first asserting that adding >0 numbers get returned when calling `AMP_Post_Meta_Box::get_featured_image_dimensions()`, and then that again that when ≤0 numbers are returned by these filters that `AMP_Post_Meta_Box::get_featured_image_dimensions()` returns the defaults.